docs/additional_examples.md

```diff
@@ -0,0 +1,212 @@
+# Additional Examples
+Examples come from [RCSB PDB Data API documentation](https://data.rcsb.org/#examples)
+
+### Entries
+Fetch information about structure title and experimental method for PDB entries:
+```
+{
+  entries(entry_ids: ["1STP", "2JEF", "1CDG"]) {
+    rcsb_id
+    struct {
+      title
+    }
+    exptl {
+      method
+    }
+  }
+}
```

Comment:
Also, I would recommend moving the GraphQL query to _after_ the Python examples. You could say something like, 

"Fetch information about structure title and experimental method for PDB entries:"
<python example>
"The above code performs the same query as the following GraphQL query:"
<GraphQL query>